nodejs/week1/homework/avg.js

```diff
@@ -0,0 +1,28 @@
+// "use strict";
```

Comment:
It's better not to leave commented code when committing to the repository :)

Reply:
done